brainscore/benchmarks/__init__.py

```diff
@@ -217,6 +217,17 @@ def _evaluation_benchmark_pool():
             # use lambda parameter-binding to avoid `benchmark_ctr` being re-assigned in the next loop iteration
             lambda benchmark_ctr=benchmark_ctr: benchmark_ctr())
 
+    # Baker 2022
+    from .baker2022 import Baker2022AccuracyDeltaFrankenstein
+    pool['kellmen.Baker2022-accuracy_delta_frankenstein'] = LazyLoad(Baker2022AccuracyDeltaFrankenstein)
+
+    from .baker2022 import Baker2022AccuracyDeltaFragmented
+    pool['kellmen.Baker2022-accuracy_delta_fragmented'] = LazyLoad(Baker2022AccuracyDeltaFragmented)
+
+    # Baker 2022
+    from .baker2022 import Baker2022InvertedAccuracyDelta
+    pool['kellmen.Baker2022-inverted_accuracy_delta'] = LazyLoad(Baker2022InvertedAccuracyDelta)
```

Comment:
why `kellmen`? I don't see that name on the paper. I actually think it might make most sense to just start with `Baker2022` here without the prefix (planning to drop that soon anyway).